Cirumflex_Combined.ino

```diff
@@ -1,12 +1,19 @@
 //Fixed colors set as globals. 03-Pattern fades between them.
 const int RANDOMnoisePIN = A5; // for random seed
 const int BRIGHTNESS = 1;      // Value pot pin number
-const int SLOWNESS = 3;        // Pot pin number; 0 is fast, 1023 is slow. 
+const int SLOWNESS = 3;        // Pot pin number; 0 is fast, 1023 is slow.
+
+// Which LED interface?
+#include "PWM_NativePin.h"
```

Comment:
#include   is...

Reply:
`#include` means exactly:
* go find the file mentioned
* cut and paste it EXACTLY HERE

Reply:
Thanks for including this private library.